cypress/integration/shared.js

```diff
@@ -114,17 +114,18 @@ export const generateIso = (sshPubKey) => {
   // click to download the discovery iso
   cy.get('#button-download-discovery-iso').click();
   // see that the modal popped up
-  cy.get('h1#pf-modal-part-8').should('be.visible');
+  cy.get('#pf-modal-part-6').should('be.visible');
   // feed in the public ssh key
-  pasteText(cy, '#form-input-sshPublicKey-discovery-field', sshPubKey);
```

Comment:
Why do you use type() instead of the pasteText method? Does this work reliably?

Reply:
From some reason, after using this pasteText method (only here), outputs an empty field, and cause a failure. 
Started to look into it and it has issue with the {backspace}, although in all other areas when using this method it is working fine. Didnt want to waist more time to investigate as i assume the UI will change, again. WDYT?

Reply:
I see what you mean. Might due to the new ability to drag a file in.